lib/models/invitation.py

```diff
@@ -0,0 +1,68 @@
+from dataclasses import dataclass
+from typing import Optional
+
+from requests import HTTPError
+
+from lib.api.checkr_api import CheckrAPI
+from lib.helpers.checkr_api_error import CheckerAPIFailedCreation
+
+
+class WorkLocation:
```

Comment:
I think I created it and then forgot about it and just used dict. It may be better to keep using dict.

Reply:
These wrapper classes are for the benefit of the *user of the library* not the *implementer*. 

Could you imagine situations where you would want to use a wrapper class for work location? If so, make the wrapper class. If not, a dict is fine, but delete the unused class before merging.

Reply:
Alright! I think we can move on with dict for now, if we feel the need of a wrapper class I can add it later.